packages/elements/src/collapse/index.ts

```diff
@@ -195,13 +195,22 @@ export class Collapse extends BasicElement {
    */
   protected render (): TemplateResult {
     return html`
-      <ef-header part="header" level="${this.level}" @tap="${this.handleTap}">
-        <ef-icon icon="right" slot="left" part="toggle"></ef-icon>
+      <ef-header
+        id="header"
+        part="header"
+        level="${this.level}"
+        @tap="${this.handleTap}"
+        tabindex="0"
+        role="button"
+        aria-expanded=${this.expanded}
+        aria-controls="content"
+      >
+        <ef-icon icon="right" slot="left" part="toggle" aria-hidden="true"></ef-icon>
         <slot slot="left" name="header-left"></slot>
         <slot slot="right" name="header-right"></slot>
         ${this.header}
       </ef-header>
-      <div part="content">
+      <div id="content" part="content" role="region" aria-labelledby="header" ?hidden=${!this.expanded}>
```

Comment:
NVDA did not announce collapse/expanded state once it was expanded, It seem to have a problem `hidden`. Found on Chrome.

Reply:
@Jidapa-Pai, this is the one I was trying to solve; it works in Firefox but doesn't work in Chromium (e.g. Chrome, MS Edge).  I already try to use CSS `display: none` to hide the content but got the same result. Do you have any ideas?

Reply:
Is the `hidden` using for skip the content region when collapse? If so, can we try `tabindex`?